mongo/session.go

```diff
@@ -44,20 +44,51 @@ type sessionContext struct {
 type sessionKey struct {
 }
 
-// Session is the interface that represents a sequential set of operations executed.
-// Instances of this interface can be used to use transactions against the server
-// and to enable causally consistent behavior for applications.
+// Session is an interface that represents a MongoDB logical session. Sessions can be used to enable causal consistency
+// for a group of operations or to execute operations in an ACID transaction. A new Session can be created from a Client
+// instance. A Session created from a Client must only be used to execute operations using that Client or a Database or
+// Collection created from that Client. Custom implementations of this interface should not be used in production. For
+// more information about sessions, and their use cases, see https://docs.mongodb.com/manual/reference/server-sessions/,
+// https://docs.mongodb.com/manual/core/read-isolation-consistency-recency/#causal-consistency, and
+// https://docs.mongodb.com/manual/core/transactions/.
+//
+// StartTransaction starts a new transaction configured with the given options on this session. This method will return
```

Comment:
Surround "configured with the given options" with commas

Reply:
done.